pkg/cluster-runtime/installer.go

```diff
@@ -118,71 +124,126 @@ func getWorkerIPList(infraDriver infradriver.InfraDriver) []net.IP {
 	return workers
 }
 
-func (i *Installer) Install() (registry.Driver, runtime.Driver, error) {
-	master0 := i.infraDriver.GetHostIPListByRole(common.MASTER)[0]
-	masters := i.infraDriver.GetHostIPListByRole(common.MASTER)
-	workers := getWorkerIPList(i.infraDriver)
-	all := append(masters, workers...)
+func (i *Installer) Install() error {
+	var (
+		masters    = i.infraDriver.GetHostIPListByRole(common.MASTER)
+		master0    = masters[0]
+		workers    = getWorkerIPList(i.infraDriver)
+		all        = append(masters, workers...)
+		image      = i.infraDriver.GetClusterImageName()
+		launchCmds = i.infraDriver.GetClusterLaunchCmds()
```

Comment:
if these codes not only for install cluster,  it's prefer to use `clusterLaunchCmds` instead of `launchCmds`

and so do with image